src/ui/src/components/forms/LoginForm.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { useFormik } from 'formik';
+import { TextField } from '@material-ui/core';
+import { ILoginFormProps } from '../../interfaces/Forms/ILoginFormProps';
+import { ILoginFormValues } from '../../interfaces/Forms/ILoginFormValues';
+import validateLoginForm from '../../Validation/validateAuthForms/validationLoginForm';
+import GoogleIcon from '../../images/Google.svg';
+
+function LoginForm({
+  onSubmit,
+  toggleForm,
+  handleGoogleSignIn,
+}: ILoginFormProps) {
+  const [isClicked, setIsClicked] = useState(false);
+  const handleButtonClick = () => {
+    setIsClicked(true);
+    setTimeout(() => {
+      setIsClicked(false);
+    }, 200);
+  };
+  const formik = useFormik<ILoginFormValues>({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validate: validateLoginForm,
+    onSubmit: (values) => {
+      onSubmit(values);
+    },
+  });
+  const emailError = formik.touched.email && formik.errors.email;
+  const passwordError = formik.touched.password && formik.errors.password;
+  return (
+    <form
+      onSubmit={formik.handleSubmit}
+      className="border rounded-lg flex flex-col items-center justify-center bg-white h-4/6 w-2/4 gap-4 max-w-xl"
+    >
+      <h1 className="text-black text-center text-lg not-italic font-semibold uppercase">
+        Login
+      </h1>
+      <div className="w-9/12">
```

Comment:
What is the reason for field duplication here whereas in RegistrationForm we Map fields?
Moreover, a separate component is possible